docs/standards/hcs-2.md

```diff
@@ -136,6 +155,33 @@ example usage:
 }
 ```
 
+#### Migrate
+
```

Comment:
We should also add in the explaination that you can't infinite loop, and hcs-2 registries should only go 3 layer's deep, with no ability to reference the same topic id multiple times within the chain.